app/views/items/index.html.erb

```diff
@@ -1,18 +1,18 @@
 <div class="container-fluid">
   <h1>Item Index</h1>
   <table class="table table-striped text-left table-bordered">
-      <thead>
-        <th>Title</th>
-        <th>Description</th>
-        <th>Price</th>
-      </thead>
-<tbody>
-  <tr><% @items.each do |item| %></tr>
-    <td><%= item.title %></td>
-    <td><%= item.description %></td>
-    <td><%= item.price %></td>
-    <td><%= link_to image_tag(item.image), item_path(item) %></td>
-  <% end %>
-</tbody>
+    <thead>
```

Comment:
@Boots2291 ya you're right it was the old html.

Cool I'll merge it and then master will be up to date